python-package/sample/connect_aci.py

```diff
@@ -81,17 +108,69 @@
 # - The JKUs array carries all allowed JWKS URLs, or none if the JKU claim in
 #   the AAS JWT token sent by the server during the aTLS handshake should not
 #   be checked.
-validator = AzAasAciValidator(policies=policies, jkus=jkus)
+validator = AciValidator(policies=policies, jkus=jkus)
+
+# Parse provided headers, if any.
+headers: Mapping[str, str] = {}
+if args.headers is not None:
+    raw = args.headers.read()
+    headers = ast.literal_eval(raw)
+
+# Read in the provided body, if any.
+body: Optional[str] = None
+if args.body is not None:
+    body = args.body.read()
+
+
+def use_direct() -> None:
+    # Set up the aTLS context, including at least one attestation document
+    # validator (only one need succeed).
+    ctx = ATLSContext([validator])
+
+    # Set up the HTTP request machinery using the aTLS context.
+    conn = HTTPAConnection(args.server, ctx, args.port)
+
+    # Send the HTTP request, and read and print the response in the usual way.
+    conn.request(
+        args.method,
+        args.url,
+        body,
+        headers,
+    )
+
+    response = conn.getresponse()
+    code = response.getcode()
+
+    print(f"Status: {code}")
+    print(f"Response: {response.read().decode()}")
+
+    conn.close()
+
+
+def use_requests() -> None:
+    session = requests.Session()
```

Comment:
nit: possibly low priority but better to add this as pytest.

Reply:
I'd like to add unit tests that run automatically but we need an ACI-capable server instance. I do want to add tests for things that can be tested without a server-side component, though; that's on my to-do list.